src/deprecated/Tribe__Events__Customizer__Text.php

```diff
@@ -1,4 +1,6 @@
 <?php
+_deprecated_file( __FILE__, 'TBD' );
```

Comment:
We should suggest what to use in place of this file if we can.

Reply:
Added the alternative to the class where the feature moved too.